src/clone.d

```diff
@@ -72,11 +72,6 @@ extern (C++) StorageClass mergeFuncAttrs(StorageClass s1, FuncDeclaration f)
 
 /*******************************************
  * Check given aggregate actually has an identity opAssign or not.
- * Params:
- *      ad = struct or class
- *      sc = current scope
- * Returns:
- *      if found, returns FuncDeclaration of opAssign, otherwise null
```

Comment:
I've just used the "revert" button in the PR page, so all changes will be merely reverted.